.changelog/10381.txt

```diff
@@ -0,0 +1,3 @@
+```release-note:bug
+xds: ensure that dependent xDS resources are reconfigured during primary type warming
```

Comment:
IIUC we only introduced this bug in 1.10 betas right? Should we make that clear here - this isn't fxing a bug from 1.9.x or earlier right?

Reply:
Maybe I could change the message to be something like `xds: (beta-only) ...` so that when we generate the flattened GA changelog we just omit the beta-only messages?